Core/src/TrackFinding/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ target_sources(
     FasTrackConnector.cpp
     RoiDescriptor.cpp
     AmbiguityTrackClustering.cpp
-)
+)
```

Comment:
```suggestion
)
```

Reply:
Thank you for reveiwing, sorry this is not letting me commit the change as it says there is no change to the code